spec.html

```diff
@@ -5570,6 +5571,23 @@ <h1>Strict Equality Comparison</h1>
   <emu-clause id="sec-operations-on-objects">
     <h1>Operations on Objects</h1>
 
+    <emu-clause id="sec-allocatebasicobject" aoid="AllocateBasicObject">
+      <h1>AllocateBasicObject ( _internalSlotsList_ )</h1>
```

Comment:
I'm happy with using `Make` as the verb, but I rather like `Basic`, contrary to @zenparsing and @allenwb. I think `Default` suffers from confusion with `Ordinary`.

I propose `MakeObjectSkeleton`? By saying "object skeleton" instead of "skeleton object", hopefully the sense is conveyed that more must be done before it is usable.

Reply:
> I think `Default` suffers from confusion with `Ordinary`.

Well, that's actually a fair confusion, given that the object it returns qualifies as ordinary (as long as you include `[[Prototype]]` and `[[Extensible]]` in `_internalSlotsList_`). But I understand the desire to downplay that fact.

> I propose `MakeObjectSkeleton`?

To me, "object skeleton" suggests that the caller must "add flesh" to it in order to obtain an actual object. But that's not true: the thing returned by `AllocateBasicObject/MakeObjectSkeleton` *is* an actual object.

> I rather like `Basic`

I'd probably be okay with `MakeBasicObject`. (Other possibilities: `MakeCoreObject` or `MakeStarterObject`, but these are subject to the same qualm that @zenparsing had about `Basic`.)

Reply:
Fair enough about "skeleton", I'll change to MakeBasicObject.